includes/core.php

```diff
@@ -168,3 +169,17 @@ function script_loader_tag( $tag, $handle ) {
 function add_manifest() {
 	echo "<link rel='manifest' href='" . esc_attr( TENUP_SCAFFOLD_TEMPLATE_URL . '/manifest.json' ) . "' />";
 }
+
+/**
+ * Add SVG sprite definitions to footer.
+ */
+function svg_sprite() {
+
+	// Define SVG sprite file.
+	$svg_sprite = get_template_directory() . '/dist/svg/svg-sprite.svg';
```

Comment:
I wonder if we need to introduce the ability to filter this file. Is there a case where we would be pulling an .svg from an external source?